modules/phase_field/doc/content/modules/phase_field/ICs/MPS.md

```diff
@@ -0,0 +1,94 @@
+# Use Maximal Poisson-Disk Sampling (MPS) to generate grain centroids
+
+Random close-packing (RCP) structure can be realized by a spatial sampling process known as Maximal Poisson-Disk Sampling (MPS). For the RCP structure, the average aspect ratio of each Voronoi cell is approximately one. The RCP Voronoi structure provides an equiaxed grain structure.
+
+The MPS algorithm is described in [cite:Ebeida_2012] and it is implemented in [MeshingGenie](https://github.com/trilinos/Trilinos/tree/master/packages/meshinggenie) package in [Trilinos](https://trilinos.org).
```

Comment:
For clarity, I'd suggest to add at the end that MeshingGenie will generate a list of grain centroids as a text file that can then be read into the PolycrystalVoronoi initial condition.

Reply:
I will make the change :-)